### PR TITLE
Utils: Capitalized

### PR DESCRIPTION
**Pull request by peter604**
Hey there!

Stumbled upon this little gem and would like to file a feature request for "capitalization". I took the liberty of whipping up the necessary bits and see if you could kindly review and publish this. Of course, any feedback is welcomed. 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,4 +71,4 @@ injected:
 [fn-join]: https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/documentation/join.md
 [fn-split]: https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/documentation/split.md
 [fn-ternary]: https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/documentation/ternary.md
-[fn-switch]: https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/documentation/switch.md
\ No newline at end of file
+[fn-switch]: https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/documentation/switch.md
```
Comment (icarus-sullivan) [inline README.md:74]: Was this a copy paste error?

File: documentation/capitalized.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[Home](https://github.com/icarus-sullivan/serverless-plugin-utils/blob/master/README.md)
+
+# fn::lower
```
Comment (icarus-sullivan) [inline documentation/capitalized.md]: Copy paste error,  seems this has comments from fn::lower.

File: src/utils/capitalized.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ['fn::capitalized']: (value) => value.charAt(0).toUpperCase() + value.slice(1).toLowerCase(),
```
Comment (icarus-sullivan) [inline src/utils/capitalized.js]: Does capitalize also enforce lowercasing the 1-end string? Might be good to err on the side of caution and not add it unless other languages/frameworks do the same.

**Review by icarus-sullivan — CHANGES_REQUESTED** [review]

**Comment by icarus-sullivan** [comment]
Hey @peter604 ,

Had some comments about the PR, appreciate the additinal idea.

**Comment by peter604** [comment]
Thanks for the feedback! I've removed the lowercase bit, and thanks for spotting the copy/paste... 🤦

**Comment by icarus-sullivan** [comment]
Thanks again for the addition, your util should now be present in version 0.1.1

**Pull request merged**